TheThingsNetwork/lorawan-stack
# Reimplement move away prompt in payload formatter views

#### Summary


Closes #6384 

#### Changes


- Use `unstable_useBlocker`
- Update ConsoleApp to use `RouterProvider`
- Refactor Payload Formatters Form
- Add move away prompt container

<img width="996" alt="Screenshot 2023-08-01 at 14 43 46" src="https://github.com/TheThingsNetwork/lorawan-stack/assets/56658938/2e0ed0c1-8a67-4fa3-9318-52232580be51">

#### Testing



1. Log in in Console.
2. Go to any device payload foratter.
3. Choose Formatter type - Custom Javascript formatter.
4. Change something in the code editor.
5. Go to overview or any other place, so move away from the payload formatter.
6. See a prompt that warns you about the unsaved changes.

#### Notes for Reviewers


Using an unstable version of the useBlocker until react-router brings back `usePrompt` and `useBlocker`.

#### Checklist
